docs/source/distributed.rst

```diff
@@ -395,6 +395,24 @@ of 16
 .. autofunction:: all_gather_multigpu
 
 
+Third-party backends
+--------------------
+
+Besides GLOO/MPI/NCCL backends, Pytorch distributed supports third-party backends by
```

Comment:
* "Besides the Gloo/MPI/NCCL..."
* "PyTorch"
* "by" -> "through a"

Reply:
amended